cassandra/cqlengine/models.py

```diff
@@ -492,7 +494,10 @@ def column_family_name(cls, include_keyspace=True):
     def _raw_column_family_name(cls):
         if not cls._table_name:
             if cls.__table_name__:
-                cls._table_name = cls.__table_name__.lower()
+                if cls.__table_name_case_sensitive__:
+                    cls._table_name = cls.__table_name__
+                else:
+                    cls._table_name = cls.__table_name__.lower()
             else:
```

Comment:
Agreed on explicit table name. I was just going down the path of making model names explicit for simplicity. In your example above, I would encourage users to just name the model ``automobile_garage`` in the first place, instead of having the mapper transform the name.

In any case, I don't feel too strongly about it. We're going to discuss this today internally. I will also ping OP on the ticket to make sure it addresses their issue.